cpp/include/raft/matrix/math.cuh

```diff
@@ -302,6 +302,20 @@ void ratio(
 
 /** @} */
 
+/**
+ * @brief Argmin: find the row idx with minimum value for each column
+ * @param in: input matrix
+ * @param n_rows: number of rows of input matrix
+ * @param n_cols: number of columns of input matrix
+ * @param out: output vector of size n_cols
+ * @param stream: cuda stream
+ */
+template <typename math_t, typename out_t, typename idx_t = int>
+void argmin(const math_t* in, idx_t n_rows, idx_t n_cols, out_t* out, cudaStream_t stream)
```

Comment:
We recently deprecated math.cuh so that we can split these files out and make the primitives easier for users to find. Let's add these to argmin.cuh and argmax.cuh